tests/cypress/support/commands/registration.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-namespace */
+/// <reference types="cypress" />
+
+declare namespace Cypress {
+  interface Chainable {
+    /**
+     * Custom command to check if all form inputs are not in error state.
+     * @param form Form name.
+     * @param errorBorderFields All fields to check.
+     * @example cy.checkFormFieldsNotInErrorStateCy('register-form')
+     */
+    checkFormFieldsNotInErrorStateCy(form: string, errorBorderFields: string): Chainable<any>
+
+    /**
+     * Custom command to submit form.
+     * @param form Form name.
+     * @example cy.submitFormCy('register-form')
+     */
+    submitFormCy(form: string): Chainable<any>
+
+    /**
+     * Custom command to hide navbar.
+     * @param device Device type.
+     * @example cy.hideNavbarCy()
+     */
+    hideNavbarCy(device: string): Chainable<any>
+
+    /**
+     * Custom command to check if 2FA page is working as expected.
+     * @param emailHash Email hash.
+     * @param formContainer Form container.
+     * @example cy.check2FAPage(emailHash, 'registration-container')
+     */
+    check2FAPage(emailHash: string, formContainer: string): Chainable<any>
+  }
+}
+
+Cypress.Commands.add('checkFormFieldsNotInErrorStateCy', (form, errorBorderFields) => {
+  cy.dataCy(form).then((form) => {
+    cy.wrap(Cypress.$(errorBorderFields, form)).should(
+      'not.have.class',
+      'border-negative-700',
+    )
+  })
+})
+
+Cypress.Commands.add('submitFormCy', (form) => {
+  cy.dataCy(form).then((form) => {
+    cy.wrap(Cypress.$('button[type=submit]', form)).click()
+  })
+})
+
+Cypress.Commands.add('hideNavbarCy', (device) => {
```

Comment:
```suggestion
Cypress.Commands.add('hideNavbar', (device) => {
```
Please remove all "Cy" from the command: commandCy